python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -452,10 +453,19 @@ async def call_pipetask(self, image_type, exposure_ids_dict):
         else:
             raise RuntimeError("Nonvalid instrument name: {self.instrument_name")
 
+        # Use the camera-agnostic yaml file if the camera-specific
+        # file does not exist.
+        cp_pipe_dir = os.environ.get("CP_PIPE_DIR")
+        pipeline_yaml_file = f"{cp_pipe_dir}/pipelines/"+f"{pipeline_instrument}/{pipe_yaml}"
```

Comment:
This might be better using `lsst.utils.getPackageDir` and `os.path.join`:
```
from lsst.utils import getPackageDir
cp_pipe_dir = getPackageDir("cp_pipe")
pipeline_yaml_file = os.path.join(cp_pipe_dir, "pipelines", pipeline_instrument, pipe_yaml)
```